packages/ui/components/pages/Fuse/Modals/PoolModal/AmountSelect.tsx

```diff
@@ -240,6 +242,15 @@ const AmountSelect = ({
     }
   };
 
+  const updateAvailableToWithdraw = async () => {
```

Comment:
Should be wrapped in a `useCallback()` hook to prevent the function from beeing created on every render.

https://reactjs.org/docs/hooks-reference.html#usecallback

Reply:
Fixed

Reply:
fixes look good to me! tested & working